contrib/get-authors

```diff
@@ -22,8 +22,16 @@ while [ $# -gt 0 ]; do
   f=$1
   shift
   contributors=
+  has_header=`grep " \*\* Top contributors" $f`
+  if [ "x$has_header" == "x" ]
+  then
+    header_lines=1
+  else
+    header_lines=`grep "\*\*\/" $f -m 1 -n | cut -d ':' -f 1`
+    if [ "x$header_lines" == "x" ]; then header_lines=1; fi
```

Comment:
Nit: header_lines is the last line of the header, so git blame will include the last line of the header. The else case of this statement should probably add 1 to header_lines.